docs/book/overview.md

```diff
@@ -0,0 +1,57 @@
+# Overview
+
+The driver consists on 2 main components
+
+- vSphere CSI Driver
+
+- Syncer
+  - Metadata Syncer
+  - Full Sync
+  - CnsOperator
+
+## vSphere CSI Driver
+
+The vSphere Container Storage Interface (CSI) Driver provides a CSI interface used by Container Orchestrators to manage the lifecycle of vSphere volumes.
+
+CSI driver is responsible for volume provisioning, attaching and detaching the volume to VMs, mounting, formatting and unmounting volumes from the pod within the node VM etc.
+
+## Syncer
+
+### Metadata Syncer
+
+The syncer is responsible for pushing PV, PVC and Pod metadata to CNS.
+These metadata will be shown in CNS dashboard (aka CNS UI) to help vSphere admin to get insights into which Kubernetes cluster(s), which app(s), Pod(s), PVC(s), PV(s) are using the volume.
+
+### Full Sync
+
+Full sync is responsible for keeping CNS up to date with Kubernetes volume metadata information like PV, PVC and Pod etc. Some cases where full sync would be helpful are listed below:
+
+- CNS goes down
+- CSI Pod goes down
+- API server goes down / K8s core services goes down
+- VC is restored to a backup point
+- etcd is restored to a backup point
+
+### CnsOperator
+
+CnsOperator is used in the context of Project Pacific. CnsOperator is responsible for
+
+- attach/detach volumes to Tanzu K8S Grid VMs
+- pushing Tanzu K8S Grid PV, PVC and Pod metadata to CNS
+
+## vSphere CSI driver supports various flavors
+
+- Vanilla Kubernetes
+
+  - Support for block volumes
```

Comment:
Can we explain here different flavors here - what is Vanilla Kubernetes, Project Pacific and Tanzu K8S Grid, instead of listing what we support in each flavor.

I think @manojvs157 is putting up Supported Feature Matrix in `docs/book/supported_features_matrix.md` for what is supported each flavor.